### PR TITLE
Added 2 new features, alt_ files, and updated ngucon

### PR DESCRIPTION
**Pull request by jvhaggard**
I added 2 new features around questing that you can use to check for quest complete and move to itopod.  I don't have the full automation done yet.  I also added 2 alt_ files that help with clicks on Windows Server systems as the standard ones don't work.  I also updated ngucon for Quest menu and coords.

### CODE REVIEW

**Comment by jvhaggard** [comment]
The alt_ files are a pain to keep separate so I wanted to go ahead and put them in so if someone wants to use a windows VM they can get that capability easily.

**Comment by jvhaggard** [comment]
anddd I forgot to squash.

File: Python/Scripts/classes/features.py [hunk 1]
```diff
@@ -748,3 +758,12 @@ def transform_slot(self, slot, threshold=0.8, consume=False):
 
         if coords:
             self.ctrl_click(*slot)
+
+    def quest_complete(self):
+        """Check if quest is complete and go to ITOPOD if it is"""
```
Comment (kujan) [inline Python/Scripts/classes/features.py]: This description should be changed because it doesn't enter ITOPOD currently.

**Comment by jvhaggard** [comment]
Aged pull request

**Pull request closed without merging**